recipe/meta.yaml

```diff
@@ -32,5 +34,7 @@ about:
 
 extra:
     recipe-maintainers:
-        - pelson
+        - bjlittle
+        - dkillick
```

Comment:
Ping @bjlittle and @dkillick. Is it OK to list you both as maintainers of `cf_units` in `conda-forge`?